src/main/kotlin/no/nav/familie/ef/sak/behandlingsflyt/steg/BeregnYtelseSteg.kt

```diff
@@ -123,6 +129,18 @@ class BeregnYtelseSteg(private val tilkjentYtelseService: TilkjentYtelseService,
                                                                    samordningsfradragType = vedtak.samordningsfradragType))
     }
 
+    private fun opprettTilkjentYtelseForSanksjonertBehandling(vedtak: Sanksjonert,
+                                                              behandling: Behandling,
+                                                              aktivIdent: String) {
+
+        val andelerTilkjentYtelse: List<AndelTilkjentYtelse> = andelerForSanksjonertRevurdering(behandling, vedtak.periode.tilPeriode())
```

Comment:
Hvis man ikke har med typen her så får det plass på 1 rad
```suggestion
        val andelerTilkjentYtelse = andelerForSanksjonertRevurdering(behandling, vedtak.periode.tilPeriode())
```

Reply:
Fixed i [87b4db9](https://github.com/navikt/familie-ef-sak/pull/1172/commits/87b4db979f71a0b7180acc19e44fb3898e683a69)